manifests/web.pp

```diff
@@ -275,24 +275,9 @@
   if $manage_resources {
     # Determine correct zabbixapi version.
     case $zabbix_version {
-      '2.2': {
-        $zabbixapi_version = '2.2.2'
-      }
-      '2.4': {
-        $zabbixapi_version = '2.4.4'
-      }
-      '3.2' : {
-        $zabbixapi_version = '3.2.1'
-      }
-      '3.4' : {
-        $zabbixapi_version = '4.0.0'
-      }
       '4.0': {
         $zabbixapi_version = '4.2.0'
       }
-      '4.4': {
```

Comment:
This suggests 4.4 is no longer supported but 4.0 is. Is that correct?

Reply:
yep. 4.0 is the supported LTS. additionally supported are 5.0 and 5.2